server/src/main/java/org/opensearch/index/translog/TranslogManager.java

```diff
@@ -119,4 +119,10 @@ public interface TranslogManager {
      * Checks if the translog has a pending recovery
      */
     void ensureCanFlush();
+
+    /**
+     *
+     * @param seqNo : operations greater or equal to seqNo should be persisted
+     */
+    void minSeqNoRequired(long seqNo);
```

Comment:
The name suggest a get operation which would return min seq number. Please change it to denote the `set` part.

Also, `required` does not provide much context around why it is required and what the caller needs to pass.

Reply:
Added javadoc to give some context . Let me know if it doesn't suffice.

Reply:
Can we incorporate it in the name itself? As I understand, we are setting a mini sequence number that should not be trimmed, right? Can the name be `minSeqNumberNotToTrim` or `minSeqNumberToKeep`?

Also, like `Translog` class, can we rename this method too so that it will be prefixed with `set`?